src/__test__/components/plots/styling/MarkerGeneSelection.test.jsx

```diff
@@ -55,6 +55,7 @@ describe('MarkerGeneSelection', () => {
 
   it('Should show the number of marker genes input', async () => {
     const mockConfig = _.merge(
+      {},
```

Comment:
what's the point of merging something empty? is the whole `_.merge` is needed at all?

Reply:
`_.merge` modifies the first object that's passed to it, and also returning the modified object. If i do `_.merge(a, b)`, the object `a` is modified with changes in `b`. If I do `_.merge({}, a, b)`, a is not modified, and the function returns a new object that is a combination of `a` and `b`.

In this case, I don't want to modify the `initialPlotConfigStates.dotPlot`, but I want an object that contains `initialPlotConfigStates.dotPlot` and `{ useCustomGenes: false }`. It is like doing 
      
```
{
   ...initialPlotConfigStates.dotPlot,
   useCustomGenes: false
}
```

Reply:
and why you didn't use? it looks cleaner and easier to read, is there any drawback?
```
{ ...initialPlotConfigStates.dotPlot, useCustomGenes: false }
```